roles/prereqs/tasks/main.yml

```diff
@@ -24,11 +31,15 @@
 
     # Setup Yum repos
     - include: yum.yml
-  when: ansible_os_family == 'RedHat'
+  when:
+    - ansible_os_family == 'RedHat'
+    - got_rke2_install_ball.stat.exists == false
```

Comment:
I think this breaks non-airgap installs. My test never installs rke2-server when not providing the tar